### ISSUE TITLE
Bug with "preventOtherCase: false"

### ISSUE THREAD

**Issue by Roper89** [AuthMe/AuthMeReloaded#1415]
Bug with 
```
# Do we need to prevent people to login with another case?
# If Xephi is registered, then Xephi can login, but not XEPHI/xephi/XePhI
    preventOtherCase: false
```
XEPHI/xephi/XePhI — **Different accounts are tied to one location.**
Players use this bug to find someone else's house.

Players specifically create the same account with a modified register to appear on a someone else's property and there to harm.

**Comment by ljacqu** [comment]
We should remove `preventOtherCase: false`. I'm surprised it still halfway works, and afaik there's no way to prevent bugs since other plugins don't take this possibility into account.

Do you need this setting for something?

**Comment by sgdc3** [comment]
@ljacqu what do you mean? Always turn on the preventOtherCase feature?

**Comment by sgdc3** [comment]
@ljacqu ?

**Comment by Roper89** [comment]
> Do you need this setting for something?

I use "`preventOtherCase: false`" because some players forget under what register they register and then can not get on the server.

**Comment by ljacqu** [comment]
I have to admit I'm not sure what the setting exactly does—if `true` it only allows a player to join with the registered casing, OK. But when it's `false`, does it allow multiple accounts to be registered with different casings? Because IMHO we shouldn't allow that.

**Comment by sgdc3** [comment]
@ljacqu No, if it is false we consider sgdc3 and SGdc3 as the same user, same password.

**Comment by sgdc3** [comment]
@Roper89 In your opinion how should we handle that? What do you expect to happen?

**Comment by Roper89** [comment]
if you can not fix the connection of different players to one geolocation.
Make it impossible to register multiple accounts XEPHI/xephi/XePhI
But with the possibility of entering with the wrong register

_(Sorry, I'm using translate.google.com)_

**Comment by Twonox** [comment]
I think we don't need to allow no-sensitive case.

**Comment by sgdc3** [comment]
then we should drop the username db column usage @ljacqu

**Comment by sgdc3** [comment]
it's just a duplicate now
